packages/tracing/test/browser/request.test.ts

```diff
@@ -96,15 +96,15 @@ describe('callbacks', () => {
       const noFetchData = { args: fetchHandlerData.args, startTimestamp: fetchHandlerData.startTimestamp };
```

Comment:
It won't let me put this on the line above with the actual test name for some reason, but can this become a `'neither creates span nor adds headers if there is no fetch data in handler data'` test, using `always`, `always` rather than `always`, `never`?